atom/browser/api/atom_api_app_mac.mm

```diff
@@ -0,0 +1,63 @@
+// Copyright (c) 2013 GitHub, Inc.
+// Use of this source code is governed by the MIT license that can be
+// found in the LICENSE file.
+
+#include "atom/browser/api/atom_api_app.h"
+
+#import <Cocoa/Cocoa.h>
+
+#include "base/strings/sys_string_conversions.h"
+
+namespace atom {
+
+namespace api {
+
+#if defined(OS_MACOSX) && defined(MAS_BUILD)
```

Comment:
This check is redundant. The whole file should be excluded from the builds if it's not a MAS build.
Check the comment below how to do that.